trento/adoc/trento-personal-access-tokens.adoc

```diff
@@ -0,0 +1,70 @@
+include::product-attributes.adoc[]
+
+[[sec-trento-personal-access-tokens]]
+
+== Personal Access Tokens
+:revdate: 2025-11-05
+
+{trento} allows users to create Personal Access Tokens (PATs) for authentication and authorization purposes in third party integrations with the APIs exposed by its components:
+
+ * link:https://www.trento-project.io/web/swaggerui/[Web]
+ * link:https://www.trento-project.io/wanda/swaggerui/[Wanda]
+
+=== About Personal Access Tokens
+
+Personal Access Tokens are an alternative to using username/password pairs for authentication to {trento} APIs and are intended to access resources on behalf of the token owner.
+
+A Personal Access Token has the same capabilities to access resources and perform actions on those resources that the owner of the token has.
+
+[NOTE]
+====
+If user permissions are changed or the user is deleted or locked, the capabilities of the Personal Access Tokens owned by that user will be affected accordingly.
```

Comment:
nit:
maybe @dmpop will highlight this, but usually, we don't use future form in docs (as far as I remember).
So in this case, we would have `by that user are affected accordingly`.
There are some few more future case forms during the text